modules/check_in/app/controllers/check_in/v2/appointments_controller.rb

```diff
@@ -18,8 +18,7 @@ def index
         # make a call to facilities and clinic endpoints
```

Comment:
these comments can also be removed now that we have implemented the facilities and clinics calls